src/Adapter/Order/CommandHandler/UpdateProductInOrderHandler.php

```diff
@@ -172,9 +187,20 @@ private function assertProductCanBeUpdated(
 //            }
 //        }
 
-        //check if product is available in stock
-        if (!Product::isAvailableWhenOutOfStock(StockAvailable::outOfStock($orderDetail->product_id))) {
-            $availableQuantity = StockAvailable::getQuantityAvailableByProduct($orderDetail->product_id, $orderDetail->product_attribute_id);
+        // Check if product is available in stock
+        if (
+            !Product::isAvailableWhenOutOfStock(
+                StockAvailable::outOfStock(
+                    $orderDetail->product_id,
+                    $orderDetail->id_shop
```

Comment:
Oh I guess the main/real fix is here no?

Reply:
The same check is performed in `AddProductToOrderHandler` so it should probably be fixed as well

Reply:
As I said you should fix `AddProductToOrderHandler` while you're at it It was built with the exact same code so there are high chances that it will have the exact same bug No need to wait for QA or community to find the bug and open an issue